src/RefractDataStructure.cc

```diff
@@ -129,7 +129,7 @@ namespace drafter {
         void operator()(const ValueInfo& value, ConversionContext& context) {
 
             if (!std::get<2>(value)) {
-                context.warn(snowcrash::Warning("invalid value format for 'number' type. Valid format is as specifified in 'C' function 'strtod()' ", snowcrash::MSONError, std::get<1>(value).sourceMap));
+                context.warn(snowcrash::Warning("invalid value format for 'number' type. Valid format is as specifified in 'C language' function 'strtod()' ", snowcrash::MSONError, std::get<1>(value).sourceMap));
```

Comment:
Perhaps it would be best to say the format is found within the MSON Specification, and we add a section on what is a valid number. Not everyone that uses API Blueprint is a programmer.